enhancement(hr-portal): updated data to be in more realistic like str…

tishko0 opened the pull request:
…ucture

yradoeva commented:
1. Some of the tree nodes couldn't be collapsed, but looks like they should (they have child nodes).
<img width="1723" alt="Screenshot 2025-02-14 at 17 31 45" src="https://github.com/user-attachments/assets/76062562-0c21-438b-b553-162a3345e967" />
and 
<img width="1726" alt="Screenshot 2025-02-14 at 18 24 51" src="https://github.com/user-attachments/assets/0da6cf1b-246f-47e1-a862-480a44058237" />


2. I've noticed that some people are shown twice in the grid, but perhaps they shouldn't?
Emma Davis
<img width="571" alt="Screenshot 2025-02-14 at 17 38 21" src="https://github.com/user-attachments/assets/7d700d83-83c8-402a-88f9-9a06f1e6e742" />

Michael Brown
<img width="809" alt="Screenshot 2025-02-14 at 17 39 50" src="https://github.com/user-attachments/assets/d87c8156-d963-4a02-8b7e-23b756c838ea" />

Noah White
<img width="769" alt="Screenshot 2025-02-14 at 17 41 32" src="https://github.com/user-attachments/assets/6269df14-69a5-4fdb-bc8a-47c5c3cb2ae8" />

Sophia Green
<img width="787" alt="Screenshot 2025-02-14 at 17 43 08" src="https://github.com/user-attachments/assets/cea9a404-57f6-4a52-b1d3-884433bd1ca7" />

Olivia Nelson
<img width="933" alt="Screenshot 2025-02-14 at 18 04 55" src="https://github.com/user-attachments/assets/4bacec26-c37a-4184-a07d-4c87059341de" />

Chloe Edwards
<img width="806" alt="Screenshot 2025-02-14 at 18 09 23" src="https://github.com/user-attachments/assets/9044a055-44f0-4e89-814a-a3495a35062e" />

dkamburov commented:
@tishko0 please see comments

MarielaTihova commented:
1. Please remove all of the occurrences of this type: Person X is present both as a boss (a level above other employees) and as an employee of themselves. For example Michael Brown is both the boss and the employee in a team managed by him. We don't see such examples in the UX
2. A person can be both a regular employee working under somebody else and a boss of other employees but cases like Sophia Green being a QA lead and being under the Backend lead seem unrealistic. There are records where a Frontend Lead is under a Backend lead also. Please have a look at the data and make sure the hierarchy of employees looks natural, as it would be in a real life company
3. Data is still not enough. Most of the records, apart from the first 3-4, has just 1 or 2 people under a manager.
4. The issue #66 mentions removing the default sorting and I can still see it present here.

MarielaTihova commented:
There are employees who pay to work in the company from this demo? 😄 

Some of the employees have a negative gross salary. Please fix that. Other comments seem to be resolved

yradoeva commented:
@tishko0 1. Can you please set a different photo for these 2 guys - Jack Thompson and Henry Carter: 
<img width="1705" alt="image" src="https://github.com/user-attachments/assets/ae098f16-38d5-455e-b62a-d7e4dd53103d" />
.
<img width="667" alt="image" src="https://github.com/user-attachments/assets/554d4e97-08ce-40d3-a09a-984a63fbe8bd" />

and for these 2 ladies - Chloe Edwards & Emily Harris:
<img width="968" alt="image" src="https://github.com/user-attachments/assets/9ff9b6c3-bd87-46c8-b005-849132f5a86f" />
.
<img width="680" alt="image" src="https://github.com/user-attachments/assets/2fd83ce3-e5d9-4e87-abf8-5edc7043221b" />


2. These 3 people are still shown twice (and when you select the one row, the other gets selected as well:
Olivia Nelson
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/5f2db816-0817-46a5-9122-eae991f490aa" />

Noah White:
<img width="987" alt="image" src="https://github.com/user-attachments/assets/b4610cf5-d9c4-4a45-9602-a8f5aaa554ec" />

Sophia Green:
<img width="1016" alt="image" src="https://github.com/user-attachments/assets/1abe5ca1-1ccb-4140-9f7f-c7fa809ed913" />

tishko0 commented:
> @tishko0 1. Can you please set a different photo for these 2 guys - Jack Thompson and Henry Carter: <img alt="image" width="1705" src="https://private-user-images.githubusercontent.com/167870501/414202089-ae098f16-38d5-455e-b62a-d7e4dd53103d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4Nzc1OTgsIm5iZiI6MTczOTg3NzI5OCwicGF0aCI6Ii8xNjc4NzA1MDEvNDE0MjAyMDg5LWFlMDk4ZjE2LTM4ZDUtNDU1ZS1iNjJhLWQ3ZTRkZDUzMTAzZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxOFQxMTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZmU3NDA3NzcyNzVmNzE5YWQ1ODc1NmYxNTUyN2NiM2Q0NDgzMmYyN2M1Y2U5YTlmMGI2NjkyNGQzZDFiMDM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.T1MniCZsXkhEDY_CvSvSyv-CB6o7Srtoc6NPv1xvkc8"> . <img alt="image" width="667" src="https://private-user-images.githubusercontent.com/167870501/414202717-554d4e97-08ce-40d3-a09a-984a63fbe8bd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4Nzc1OTgsIm5iZiI6MTczOTg3NzI5OCwicGF0aCI6Ii8xNjc4NzA1MDEvNDE0MjAyNzE3LTU1NGQ0ZTk3LTA4Y2UtNDBkMy1hMDlhLTk4NGE2M2ZiZThiZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxOFQxMTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lOTY3ZWFmNzZjYmI1YWRlYjBmZDU5OWQ1OTM2ZDhkOWRlZjVkOWY2ODNhYjQ4ZTU3ZjBkYzc2NGYxMjZiMmUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hUC1w8J3FAYyPIJ8J3MUEntPvRD_RxcreH9tHKEuE2Q">
> 
> and for these 2 ladies - Chloe Edwards & Emily Harris: <img alt="image" width="968" src="https://private-user-images.githubusercontent.com/167870501/414207893-9ff9b6c3-bd87-46c8-b005-849132f5a86f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4Nzc1OTgsIm5iZiI6MTczOTg3NzI5OCwicGF0aCI6Ii8xNjc4NzA1MDEvNDE0MjA3ODkzLTlmZjliNmMzLWJkODctNDZjOC1iMDA1LTg0OTEzMmY1YTg2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxOFQxMTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YTVhNWM5Mjc5NDdiZTc4MjVmZjg4YWY3Y2FlMjljMTk0ZTdhOWZlNDFjYjg1ZmYxY2U4OGQ0ZTc2MDVkMGY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.OpUikdlQCvsDCyPRuLsx1UipkOOBkMYJEshvne-4PUs"> . <img alt="image" width="680" src="https://private-user-images.githubusercontent.com/167870501/414208259-2fd83ce3-e5d9-4e87-abf8-5edc7043221b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4Nzc1OTgsIm5iZiI6MTczOTg3NzI5OCwicGF0aCI6Ii8xNjc4NzA1MDEvNDE0MjA4MjU5LTJmZDgzY2UzLWU1ZDktNGU4Ny1hYmY4LTVlZGM3MDQzMjIxYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxOFQxMTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01OGRhODg4NzQ2ZDU3ZDUwOTQ1MTFiMDM1ZWM2Y2VlZTE3ZWQ2MGM2MDU1ZjdmNzkwNzExOTM5NWJjNDFkNGMzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.jceTBZhPb_cmeMs2WlV4Zj_I1s7beLKZn39e-kUYmo8">
> 
> 2. These 3 people are still shown twice (and when you select the one row, the other gets selected as well:
>    Olivia Nelson
> 
> <img alt="image" width="1000" src="https://private-user-images.githubusercontent.com/167870501/414212731-5f2db816-0817-46a5-9122-eae991f490aa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4Nzc1OTgsIm5iZiI6MTczOTg3NzI5OCwicGF0aCI6Ii8xNjc4NzA1MDEvNDE0MjEyNzMxLTVmMmRiODE2LTA4MTctNDZhNS05MTIyLWVhZTk5MWY0OTBhYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxOFQxMTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZTI4ZWYzMTIyNWQ5NTc0Y2Y5MDU3ZWRjYjEwM2VhYjFlYzdkMmRiNmI3ZTI4YjEwZDI2M2M5M2VhYWQ1NzlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Bu7pyCT4i3xukNG1yv_CGsElrooMAjQvuKsBDig-LOw">
> Noah White: <img alt="image" width="987" src="https://private-user-images.githubusercontent.com/167870501/414211263-b4610cf5-d9c4-4a45-9602-a8f5aaa554ec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4Nzc1OTgsIm5iZiI6MTczOTg3NzI5OCwicGF0aCI6Ii8xNjc4NzA1MDEvNDE0MjExMjYzLWI0NjEwY2Y1LWQ5YzQtNGE0NS05NjAyLWE4ZjVhYWE1NTRlYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxOFQxMTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NDZmMjBmZTliZDFjODQzMjE1NjBiZjUxOTNiZGUxZjc0Y2YzYjllNDQwNTFjNmI1YmZmMzZlYTg4MWY4OTRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.AhBSZYVBl9TR7xORIyv8cIP7ahvQ5U3G8VXkA2Q77as">
> 
> Sophia Green: <img alt="image" width="1016" src="https://private-user-images.githubusercontent.com/167870501/414212282-1abe5ca1-1ccb-4140-9f7f-c7fa809ed913.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk4Nzc1OTgsIm5iZiI6MTczOTg3NzI5OCwicGF0aCI6Ii8xNjc4NzA1MDEvNDE0MjEyMjgyLTFhYmU1Y2ExLTFjY2ItNDE0MC05ZjdmLWM3ZmE4MDllZDkxMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxOFQxMTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mOTNiNDE4MTE1ODNhMTljODhmYjg5MTZlZWJmMTIyZjcyNGFjNTk3ODRhODMyNjc0N2I3ZGIwNTgxMzQzMWRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.hqcM9bREOdfFGgGjiG7mb0OfJsbILr8Gk4_CGFovsCc">

tishko0 commented:
I have fixed the salary with negative score and removed duplicate entries but I already have more entries and than pictures for, so for now I removed the spotted duplicate pictures.

yradoeva commented:
Looks ok to me, thanks @tishko0
